packages/vaadin-login/theme/material/vaadin-login-overlay-styles.js

```diff
@@ -1,357 +1,356 @@
+      padding: 1.5rem 1.5rem 1.5rem 2.5rem;
     }
 
-    :host([theme~='with-overlay']) [part='form'] {
-      padding: 2rem 2.5rem 2rem 1.8rem;
+    :host([dir='rtl']) [part='form'] {
+      margin: 80px -20px 40px 30px;
     }
+  }
 
-    :host([theme~='with-overlay']) [part='form'] h2 {
-      text-align: center;
-      font-size: 1.8em;
-      font-weight: 500;
+  /* Landscape really big screen */
+  @media only screen and (min-width: 1440px) {
+    [part='content'] {
+      max-width: none;
+      margin: auto;
     }
 
-    :host([theme~='with-overlay']) [part='form'] ::slotted(form) {
-      display: flex;
-      flex-direction: column;
+    [part='card'] {
+      max-width: 960px;
+    }
+
+    [part='brand'] {
+      padding-left: 4rem;
+      padding-right: 1rem;
     }
 
-    /* Small screen */
-    @media only screen and (max-width: 413px) {
-      :host([theme~='with-overlay']) [part='form'] {
-        padding-top: 0.75rem;
-      }
+    [part='form'] {
+      height: 100%;
+      -webkit-overflow-scrolling: touch;
     }
 
-    /* Landscape big screen */
-    @media only screen and (min-width: 1024px) {
-      :host([theme~='with-overlay']) [part='form'] {
-        padding: 2em;
-      }
+    :host(:not([dir='rtl'])) [part='form'] {
+      margin-right: 22px;
+    }
 
-      :host([theme~='with-overlay']) [part='form'] ::slotted(form) {
-        justify-content: space-around;
-        margin-top: 20px;
-        min-height: 250px;
-      }
+    :host(:not([dir='rtl'])) [part='form'] ::slotted(*) {
+      right: 80px;
     }
 
     /* RTL styles */
-    :host([dir='rtl'][theme~='with-overlay']) [part='form'] {
-      padding: 2rem 1.8rem 2rem 2.5rem;
+    :host([dir='rtl']) [part='brand'] {
+      padding-right: 4rem;
+      padding-left: 1rem;
+    }
+
+    :host([dir='rtl']) [part='form'] {
+      margin-left: 22px;
+    }
+
+    :host([dir='rtl']) [part='form']::slotted(*) {
+      left: 80px;
+    }
+  }
+
+  /* Handle iPhone X notch */
+  @media only screen and (device-width: 375px) and (device-height: 812px) and (-webkit-device-pixel-ratio: 3) {
+    [part='card'] {
+      padding: 0 constant(safe-area-inset-bottom);
+      padding: 0 env(safe-area-inset-bottom);
     }
-  `,
-  { include: ['material-typography'], moduleId: 'material-login-overlay' }
-);
+
+    :host(:not([dir='rtl'])) [part='brand'] {
+      margin-left: calc(constant(safe-area-inset-bottom) * -1);
+      margin-left: calc(env(safe-area-inset-bottom) * -1);
+      padding-left: calc(1.5rem + constant(safe-area-inset-bottom));
+    }
+
+    /* RTL styles */
+    :host([dir='rtl']) [part='brand'] {
+      margin-right: calc(constant(safe-area-inset-bottom) * -1);
+      margin-right: calc(env(safe-area-inset-bottom) * -1);
+      padding-right: calc(1.5rem + constant(safe-area-inset-bottom));
+    }
+  }
+`;
+
+registerStyles('vaadin-login-overlay-wrapper', [overlay, typography, loginOverlayWrapper], {
+  moduleId: 'vaadin-login-overlay-wrapper-material-styles'
```

Comment:
We probably should update this later to use module id with `material-` prefix. Not a blocker.

Reply:
Let's discuss separately if we could completely remove the (Polymer dependent) `{moduleId}`'s